docs/section10/10-4.md

```diff
@@ -0,0 +1,58 @@
+# ASHVAC – Rule 10-4  
+
+**Rule ID:** 10-4  
+ 
+**Rule Description:** Baseline HVAC systems using fossil fuel shall be modeled using natural gas. Except where natural gas is not available for the proposed building site, propane shall be used as the heating fuel.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:**   
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**   
+
+1. Applicable to baseline air-side systems 3 and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_heating_fuels_p()
+
+
+**Manual Check:** None   
+ 
+
+**Rule Logic:**  
+**Applicability Check 1:**   
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
```

Comment:
I wonder if we should just go through all baseline HVAC systems irrespective of type and verify that the heating fuel source is not erroneously modeled as something other than natural gas.